package.json

```diff
@@ -171,11 +208,30 @@
       ],
       "installTestPath": "./.omnisharp/1.37.17/run",
       "platformId": "osx",
+      "isFramework": true,
       "integrity": "43D6B18F390B8ED92231044E87413831E565E9AB5938223B19CA97B3628DDDF4"
     },
     {
       "id": "OmniSharp",
-      "description": "OmniSharp for Linux (x86)",
+      "description": "OmniSharp for OSX (.NET 6)",
```

Comment:
I think you are missing a split into x64 and arm64 here

Reply:
good catch. We don't have one currently, because we install the x64 for both architectures. Since the net6 builds are currently being built as self-contained this would cause issues.

Reply:
I added profiles for the other platforms. @filipw, can you give it a try on Mac?

Reply:
I went through this list and updated our profiles based on it, from the beta83 publish of omnisharp-roslyn:

```
1.37.18-beta.83/omnisharp-win-x86.zip
1.37.18-beta.83/omnisharp-win-x86-net6.0.zip
1.37.18-beta.83/omnisharp-win-x64.zip
1.37.18-beta.83/omnisharp-win-x64-net6.0.zip
1.37.18-beta.83/omnisharp-win-arm64.zip
1.37.18-beta.83/omnisharp-win-arm64-net6.0.zip
1.37.18-beta.83/omnisharp-osx.zip
1.37.18-beta.83/omnisharp-osx.tar.gz
1.37.18-beta.83/omnisharp-osx-x64-net6.0.zip
1.37.18-beta.83/omnisharp-osx-x64-net6.0.tar.gz
1.37.18-beta.83/omnisharp-osx-arm64-net6.0.zip
1.37.18-beta.83/omnisharp-osx-arm64-net6.0.tar.gz
1.37.18-beta.83/omnisharp-mono.zip
1.37.18-beta.83/omnisharp-mono.tar.gz
1.37.18-beta.83/omnisharp-linux-x86.zip
1.37.18-beta.83/omnisharp-linux-x86.tar.gz
1.37.18-beta.83/omnisharp-linux-x64.zip
1.37.18-beta.83/omnisharp-linux-x64.tar.gz
1.37.18-beta.83/omnisharp-linux-x64-net6.0.zip
1.37.18-beta.83/omnisharp-linux-x64-net6.0.tar.gz
1.37.18-beta.83/omnisharp-linux-arm64.zip
1.37.18-beta.83/omnisharp-linux-arm64.tar.gz
```